pkgs/top-level/all-packages.nix

```diff
@@ -13002,6 +13002,7 @@ with pkgs;
 
   zabbix20 = callPackage ../servers/monitoring/zabbix/2.0.nix { };
   zabbix22 = callPackage ../servers/monitoring/zabbix/2.2.nix { };
+  zabbix34 = callPackage ../servers/monitoring/zabbix/3.4.nix { };
```

Comment:
Do you know if zabbix34 is backwards compatible? If so, I think we would want to make this the default "zabbix" attribute (the top "zabbix" is still at 1.8!)/

Reply:
@matthewbauer  Hi, thanks for taking a look into this. I know that zabbix server is compatible with zabbix agent with lower versions, (we actually have an Amazon linux zabbix server (3.4) communicating with nixos servers that has the agent version 2.2 (https://www.zabbix.com/documentation/3.4/manual/appendix/compatibility) .  For the other way around i don't believe it is the case, and even if it is the case people strongly advise against it. Not sure if it is possible to upgrade the default Zabbix server and agent to 3.4 ?

Reply:
Ok it's probably fine for now.